dpnp/fft/dpnp_utils_fft.py

```diff
@@ -66,6 +72,37 @@ def _check_norm(norm):
         )
 
 
+# TODO: c2r keyword is place holder for irfftn
+def _cook_nd_args(a, s=None, axes=None, c2r=False):
+    if s is None:
+        shapeless = True
+        if axes is None:
+            s = list(a.shape)
+        else:
+            s = numpy.take(a.shape, axes)
+    else:
+        shapeless = False
+
+    for s_i in s:
+        if s_i is not None and s_i < 1 and s_i != -1:
+            raise ValueError(
+                f"Invalid number of FFT data points ({s_i}) specified."
+            )
+
+    if axes is None:
+        axes = list(range(-len(s), 0))
+
+    if len(s) != len(axes):
+        raise ValueError("Shape and axes have different lengths.")
+
+    s = list(s)
+    if c2r and shapeless:
+        s[-1] = (a.shape[axes[-1]] - 1) * 2
```

Comment:
looks not cover by any test

Reply:
`c2r` is a place holder for backward real FFT. So, it will be covered in next PR, a TODO is already added at the top of this function for it.